cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -34,6 +34,7 @@
  * Uses GPI calls to interface to the simulator.
  */
 
+int unused;
```

Comment:
Doesn't seem to work?

Reply:
Yeah, I need to find the time to work on this further.